spec/integration/lhm_spec.rb

```diff
@@ -10,74 +10,78 @@
 
   before(:each) { connect_master! }
 
-  describe "changes" do
+  describe 'retain triggers' do
     before(:each) do
-      table_create(:users)
-    end
-
-    it "should add a column" do
-      Lhm.change_table(:users, :atomic_switch => false) do |t|
-        t.add_column(:logins, "INT(12) DEFAULT '0'")
-      end
-
       slave do
-        table_read(:users).columns["logins"].must_equal({
-          :type => "int(12)",
-          :is_nullable => "YES",
-          :column_default => '0'
-        })
+        table_create(:users)
       end
     end
 
-    it "should retain triggers while migration if retain_triggers is set to true in atomic switch" do
-      table_name = 'users'
-      trigger_name = 'timestamp_trigger'
-      check_and_create_trigger_on_users_table
-      Lhm.change_table(table_name.to_sym, :atomic_switch => true, :cleanup => true, :retain_triggers => true) do |t|
-        t.add_column(:logins, "INT(12) DEFAULT '0'")
-      end
-
+    it 'should retain triggers while migration if retain_triggers is set to true in atomic switch' do
       slave do
+        table_name = 'users'
+        trigger_name = 'timestamp_trigger'
+        check_and_create_trigger_on_users_table
+        Lhm.change_table(table_name.to_sym, :atomic_switch => true, :cleanup => true, :retain_triggers => true) do |t|
```

Comment:
can we add `Lhm.cleanup(true)` during before and after each in the test?

Reply:
done